Task byggaearlyaccess

Wessebanan opened the pull request.

Wessebanan committed:
rast

Wessebanan committed:
coola grejer i system

Wessebanan committed:
fräscha .obj:er

Wessebanan committed:
slut för dagen system.cpp ser rätt bra ut

Wessebanan committed:
Tog bort en utkommenterad grej som inte behövs.

Wessebanan committed:
Fräschare .obj:er

Wessebanan committed:
eventdata

Wessebanan committed:
Merge branch 'Task_ByggaEarlyAccess' of https://github.com/Mabaria/DV1504Grupp1 into Task_ByggaEarlyAccess

Wessebanan committed:
jag har visst pushat

Wessebanan committed:
Bytte till bättre namn på båtgrejer i system

Wessebanan committed:
merge med patcherino

Wessebanan committed:
Kompilerar igen.

Commit:
Floor2 updated

Commit:
Merge branch 'Task_ByggaEarlyAccess' of https://github.com/Mabaria/DV1504Grupp1 into Task_ByggaEarlyAccess

Wessebanan committed:
Blev så mycket i buildgraphicaluserinterface så jag flyttade till funktioner.

Wessebanan committed:
Merge branch 'Task_ByggaEarlyAccess' of https://github.com/Mabaria/DV1504Grupp1 into Task_ByggaEarlyAccess

Wessebanan committed:
lite till

Mabaria committed:
Added camera reset button

Mabaria committed:
Mergiboi

Mabaria committed:
Added button again >:I

Wessebanan committed:
PULLA LITE

Wessebanan committed:
Merge branch 'Task_ByggaEarlyAccess' of https://github.com/Mabaria/DV1504Grupp1 into Task_ByggaEarlyAccess

Wessebanan committed:
SideViewPanel fick en egen kamera

Wessebanan committed:
idfk lunch

Wessebanan committed:
La till removeevent också som behöver fyllas precis som addevent

Taumic7 committed:
fixed all warnings

Taumic7 committed:
Merge branch 'Task_ByggaEarlyAccess' of https://github.com/Mabaria/DV1504Grupp1 into Task_ByggaEarlyAccess

Commit:
Fixed 3D text and Added cap for Y (Perspective Cam Movement)

Commit:
A merge has been made

Commit:
Barely started

Commit:
Merged from master

Commit:
Merge fixed

Commit:
Controlpanel change a bit

Wessebanan committed:
merge med picking

Wessebanan committed:
local change

Wessebanan committed:
Merge branch 'Task_ByggaEarlyAccess' of https://github.com/Mabaria/DV1504Grupp1 into Task_ByggaEarlyAccess

Wessebanan committed:
merge fixed

Wessebanan committed:
intelligent intellisense

Wessebanan committed:
indexgrejre

Taumic7 committed:
change so that camera can be movable or not and fixed un intended camera position resets and what not

Wessebanan committed:
dank

Taumic7 committed:
i commit to nothing

Commit:
small change

Commit:
A fix in ps_shader

Taumic7 committed:
change reset butt on

Taumic7 committed:
mergeee

Wessebanan committed:
lite grejer

Wessebanan committed:
gick bra :^)

Wessebanan committed:
Skönare defaultkameravinkel på topview

Commit:
Added new pictures for pushed down buttons

Commit:
Merged from early access bygg

Commit:
Changed text3d pictures

Taumic7 committed:
Can now load bitmaps into panel2D and button constructor with bitmap pointer

Commit:
small changes

Taumic7 committed:
Buttons now have names and is added in constructor

Taumic7 committed:
Merge remote-tracking branch 'origin/Task_ByggaEarlyAccess' into Patch_Panel2DBitmap

Commit:
Changed names of the button png's

Commit:
Merge branch 'Task_ByggaEarlyAccess' into Task_EventMenu

Wessebanan committed:
plotting

Wessebanan committed:
Merge branch 'Task_ByggaEarlyAccess' of https://github.com/Mabaria/DV1504Grupp1 into Task_ByggaEarlyAccess

Wessebanan committed:
funkar igen

Commit:
Added
room.GetActiveEvents()

Changed:
Room.AddAutoEvent() -> Room.AddSensorEvent()

Room.AddPlotEvent() ->
Room.AddPlotterEvent() ->

Commit:
Each type of event can only be added once per room, until that type is "cleared".

Wessebanan committed:
lite handleinputgrejre

Wessebanan committed:
Merge branch 'Task_ByggaEarlyAccess' of https://github.com/Mabaria/DV1504Grupp1 into Task_ByggaEarlyAccess

Wessebanan committed:
ye

Commit:
Fixed some stuff

Commit:
Added the room pointer

Wessebanan committed:
blä

Wessebanan committed:
wololololo

Commit:
Change camera view

Commit:
Merge branch 'Task_ByggaEarlyAccess' of https://github.com/Mabaria/DV1504Grupp1 into Task_ByggaEarlyAccess

Wessebanan committed:
is good

Wessebanan committed:
is very good

Wessebanan committed:
Merge branch 'Task_ByggaEarlyAccess' of https://github.com/Mabaria/DV1504Grupp1 into Task_ByggaEarlyAccess

Commit:
Each event type is limited to one per room, until they are cleared again.
Events can be cleared correctly now.

Commit:
Merged

Wessebanan committed:
bytte från add och remove event till updateevents i system för att knappklick i menyn bara kan kalla på en funktion. Funkar att ta bort events.

Wessebanan committed:
menyn flummar mindre med sw_normal istället för sw_showdefault

Wessebanan committed:
ay

Taumic7 committed:
Added reset butt on

Taumic7 committed:
Merge branch 'Task_ByggaEarlyAccess' of https://github.com/Mabaria/DV1504Grupp1 into Task_ByggaEarlyAccess

Commit:
Boudning boxes is now not visibile if no event is present

Commit:
Created functionality to read äåö

Wessebanan committed:
allt funkar förutom att menyn visas när man startar

Wessebanan committed:
Merge branch 'Task_ByggaEarlyAccess' of https://github.com/Mabaria/DV1504Grupp1 into Task_ByggaEarlyAccess

Commit:
Merge branch 'Task_ByggaEarlyAccess' into FixaWString

Wessebanan committed:
sasa

Wessebanan committed:
Merge branch 'Task_ByggaEarlyAccess' of https://github.com/Mabaria/DV1504Grupp1 into Task_ByggaEarlyAccess

Wessebanan committed:
bajs

Wessebanan committed:
syns inte i början längre men den kör hide varje frame om inte visible

Wessebanan committed:
STOR TEXT

Taumic7 committed:
a function for all time that have passed since start in seconds

Wessebanan committed:
snyggat till inför möte

Wessebanan committed:
inga gaps i aktiv logg boi

Mabaria reviewed (commented):
System.cpp:204
System.cpp:239
System.cpp:274
TEMP

System.cpp:510
that 🤔 

System.h:10
Todo 😡 

NotificationList
Kan vi ta in fontsize i konstruktorn kanske? Lite dumt att det är 🗻 👨‍💻

Wessebanan committed:
*Tog bort add och remove event från system för de är derpecated.
*Slaktade temp-lösningen, bytte mot "däckbounds" grejen.
*Fixade den ofärdiga kommentaren.
*Tog bort todo.
*Tog bort den tomma updatetextboxes i panel2d.
*Optimerade notisobjekten (helt task)
*Kommenterade ut lite grejer som bråkade i test_panel.
*Laddar in bilderna i activelogpanel nu så att de kan användas av notisobjekten.
*La till fontsize i konstruktorer och dynamisk sizing av textboxar baserat på fontsize.
*Tog bort en massa varningar.

Mabaria reviewed (approved):
😉 🦑 🦑 🦑 🗾 🦑 🦑 🦑 👧 🦑 🦑 🦑 💦
